Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -163,3 +166,27 @@ final class PledgePaymentMethodsViewController: UIViewController {
       |> \.textAlignment .~ .center
   }
 }
+
+extension PledgePaymentMethodsViewController: PledgeAddNewCardViewDelegate {
+  func pledgeAddNewCardViewDidTapAddNewCard(_: PledgeAddNewCardView) {
+    let addNewCardViewController = AddNewCardViewController.instantiate()
+      |> \.delegate .~ self
+    let navigationController = UINavigationController.init(rootViewController: addNewCardViewController)
+    let offset = navigationController.navigationBar.bounds.height
+
+    self.presentViewControllerWithSheetOverlay(navigationController, offset: offset)
+  }
+}
+
+extension PledgePaymentMethodsViewController: AddNewCardViewControllerDelegate {
+  func addNewCardViewControllerDismissed(_: AddNewCardViewController) {
+    self.dismiss(animated: true)
```

Comment:
Do we need this or is it just to avoid the pattern of the child dismissing itself?

Reply:
We need this to dismiss the `AddNewCardViewController` when `Cancel` is tapped.